kinesis/src/main/scala/akka/stream/alpakka/kinesis/KinesisSchedulerSettings.scala

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2016-2019 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.kinesis
+
+import scala.concurrent.duration._
+
+case class KinesisSchedulerSourceSettings(bufferSize: Int, backpressureTimeout: FiniteDuration) {
```

Comment:
Settings classes should be non-case classes to support evolving better. 
See [contributor advice](https://github.com/akka/alpakka/blob/master/contributor-advice.md#evolving-apis-with-binary-compatibility)